src/functions/meta-assistant/send-card-notification/handler.ts

```diff
@@ -0,0 +1,49 @@
+import { formatJSONResponse, NotificationHandlerResponse, ValidatedEventAPIGatewayProxyEvent } from "src/libs/api-gateway";
+import { middyfy } from "src/libs/lambda";
+import { parseCardDetails } from "src/libs/parse-utils";
+import SlackUtilities from "src/meta-assistant/slack/slack-utils";
+import { ParsedBody } from "src/types/meta-assistant";
+
+/**
+ * Sends a Slack message to the notification handler
+ *
+ * @param parsedBody parsed data from Trello
+ * @returns JSON response
+ */
+const sendNotification = async (parsedBody: ParsedBody) : Promise<NotificationHandlerResponse> => {
```

Comment:
If this function is related to the memos and trello card functionality I'm wondering if it should be in the memo-management folder